composer.json

```diff
@@ -31,7 +31,7 @@
         "symfony-cmf/seo-bundle": "^2.0",
         "symfony-cmf/routing": "^2.0",
         "symfony-cmf/routing-auto-bundle": "^2.0",
-        "symfony-cmf/sonata-phpcr-admin-integration-bundle": "^1.0@dev",
+        "symfony-cmf/sonata-phpcr-admin-integration-bundle": "dev-fix_ckeditor",
```

Comment:
sandbox has a composer.lock file in version control so changing only this line does not do anything. no big deal, just saying.